src/validations/product/category.validation.js

```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
+
+const categoryValidationSchema = Joi.object({
+  name: Joi.string().required(),
+});
+const categoryValidation = async (req, res, next) => {
+  const value = categoryValidationSchema.validate(req.body, {
+    abortEarly: false,
+  });
+  if (value.error) {
+    return res.status(400).json({
+      message: 'Invalid Category provided',
+      error: value.error.message,
+    });
```

Comment:
- How do you know if the category provided is invalid if you have not specified the valid category?
- Please check how the other validations are written